hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -0,0 +1,34 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "flash_ctrl"
+  // TODO: remove the common testplans if not applicable
+  import_testplans: ["hw/dv/tools/testplans/csr_testplan.hjson",
+                     "hw/dv/tools/testplans/mem_testplan.hjson",
+                     "hw/dv/tools/testplans/intr_test_testplan.hjson",
+                     "hw/dv/tools/testplans/tl_device_access_types_testplan.hjson"]
+  entries: [
+    {
+      name: sanity
+      desc: '''**Goal**: Basic sanity test acessing a major datapath in FLASH_CTRL.
+
+            **Stimulus**: Describe the stimulus procedure.
+
+            **Checks**": Describe the self-check procedure.
+            - add bullets as needed
+            - second bullet<br>
```

Comment:
Yes, the `<br>` was added so that the following line aligns as a sub-bullet rather than a new paragraph. Md does not support this AFAIK. Do you have another solution for it?

Also, I will be rewriting this completely anyway.

Reply:
I see. sg